src/mixer/basetrackplayer.cpp

```diff
@@ -792,6 +823,31 @@ void BaseTrackPlayerImpl::slotSetTrackColor(const mixxx::RgbColor::optional_t& c
     m_pTrackColor->forceSet(trackColorToDouble(color));
 }
 
+void BaseTrackPlayerImpl::slotTrackMoveColor(int steps) {
+    if (!m_pLoadedTrack) {
```

Comment:
Just an idea to make it a bit shorter:
```suggestion
    if (!m_pLoadedTrack || steps == 0) {
```